src/main/java/org/candlepin/model/DeletedConsumer.java

```diff
@@ -75,8 +80,9 @@
     @Column(name = "principal_name")
     private String principalName;
 
-    public DeletedConsumer(String cuuid, String oid, String okey, String oname) {
+    public DeletedConsumer(String cuuid, String cname, String oid, String okey, String oname) {
```

Comment:
Don't worry about updating these old constructors. This should either be left as-is, or removed entirely.

Reply:
Sounds good to me! Updated